sn_networking/src/record_store.rs

```diff
@@ -384,6 +391,14 @@ impl NodeRecordStore {
         let record_key = PrettyPrintRecordKey::from(&r.key).into_owned();
         trace!("PUT a verified Record: {record_key:?}");
 
+        let new_distance = self
+            .local_address
+            .distance(&NetworkAddress::from_record_key(&r.key));
+
+        // set's the local distance range to be further if the new record is further
+        // This assumes the verified record is to be held by _us_...
+        self.distance_range = self.distance_range.map(|range| range.max(new_distance));
```

Comment:
As the network grows, would we not be getting records that are closer to us? So this would mean that we'll be hanging on to the older value?

Reply:
Aye, cleaned that out and moved farthest record check into an interval setter. Should be enough and prevent too much overzealous "close" checking on store etc.